CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ You can find and compare releases at the [GitHub release page](https://github.co
 
 ## 5.6.0
 
+- Add `builder` option to `@all` directive
```

Comment:
This should go in the `Unreleased` section under `### Added`, and you can add the link to this PR while you are at it.